src/components/base/BreadCrumb.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div>
+    <span v-for="(link, i) in links" :key="'link-' + link.name">
+      <span v-if="link.path && i !== links.length - 1">
+        <router-link class="plot-nav is-size-5" :to="link.path">
+          <span>{{ link.name }}</span></router-link
+        >
+        <span class="mx-4 is-size-5">/</span>
+      </span>
+      <span v-else class="plot-nav is-size-5">{{ link.name }}</span>
+    </span>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+
+const props = defineProps<{
+  path: string;
+}>();
+
+const links = computed(() => {
+  const pathParts = props.path.split("/").filter((p) => p);
```

Comment:
This filter looks like it doesn't change anything? Do we need it?

Reply:
This also happened in other places, but just wanted to pose the question here first!

Reply:
it filters out empty strings after the split

Reply:
Oh that's cool!